helper-classes/int-list-reader.cc

```diff
@@ -0,0 +1,17 @@
+// Advent of Code 2020
+
+#include "int-list-reader.h"
+
+namespace data {
+
+std::vector<int> IntListReader::GetData() {
+    std::vector<int> ret;
+    std::vector<std::string> string_list = this->GetStringListData();
```

Comment:
Nit: can be `const`. [Const correctness ](https://isocpp.org/wiki/faq/const-correctness) is very helpful.